src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile

```diff
@@ -16,29 +16,30 @@ RUN {{packageManager}} install -y findutils tar gzip
 {{/ubi}}
 
 {{#usePublicArtifact}}
-RUN cd /opt && \
-  curl --retry 8 -s -L -O https://artifacts.opensearch.co/downloads/kibana/{{artifactTarball}} && \
-  cd -
+# TODO: Update this link or remove functionality
```

Comment:
Don't need to do for this PR but for these I end up appended [RENAMEME] to the `TODO`'s so that we can track it a little bit easier. So we will end up updating all these artifact links with real links.